tileson.hpp

```diff
@@ -2660,7 +2660,7 @@ namespace tson
 				m_json = nullptr;
 				if (fs::exists(path) && fs::is_regular_file(path))
 				{
-					std::ifstream file(path.u8string());
+					std::ifstream file(path.c_str());
```

Comment:
Any reason why you haven't used `generic_string()` here compared to your other changes? Please keep in mind that the string produced should support special characters. I'm unsure if `c_str()` on a fs::path would do that directly, but I'd not be sure unless I could prove it could handle special characters.